src/helpers/helpers.core.js

```diff
@@ -25,3 +25,19 @@ export function getElementCenterPoint(element, useFinalPosition) {
   const {x, y} = element.getProps(['x', 'y'], useFinalPosition);
   return {x, y};
 }
+
+const isOlderPart = (act, req) => req > act || (act.length > req.length && act.substr(0, req.length) === req);
```

Comment:
What about storing the `parseInt` versions from line 36 into variables and then passing numbers to this function. Would be simpler logic I think

Reply:
I think it would not work if requiring 4.0.0 and the version was 4.0.0-alpha

Reply:
Good point